site/gatsby-site/src/global.css

```diff
@@ -502,12 +502,17 @@ blockquote {
 }
 
 .titleWrapper {
-  display: flex;
   flex-direction: column;
   align-items: flex-start;
-  padding-bottom: 20px;
   border-bottom: 1px solid rgb(230, 236, 241);
-  margin-bottom: 32px;
+  min-height: 5rem;
```

Comment:
Not necessary to be done on this PR, but we should be removing `styled-components` instances wherever possible.